### DISCUSSION TITLE
Icon Request: fa-brick

### DISCUSSION THREAD

**Discussion by mdolberg** [FortAwesome/Font-Awesome#21233] [Icon Requests]
I am planning to use it as a way to express construction

![brick-clip-art-379570](https://cloud.githubusercontent.com/assets/11992836/10965222/2fdb3e6a-83b4-11e5-9b98-004b9641e153.jpg)

**Comment by zibbozz** [comment]
+1 really need it for a mobile app

**Comment by dhruvparekhdp** [comment]
+1

**Comment by 3UR** [comment]
+1

**Comment by willemsteller** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🧱 [U+1F9F1](https://emojipedia.org/emoji/%F0%9F%A7%B1/) Brick

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
